### ISSUE TITLE
Accessibility: Multiple Non-UM Users

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#648]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Major

Feature: Multiple Non-UM Users

Issue notes:
After adding a file, "test_um_students_multiple_sections.csv" appears as H5, it should be p instead

Criterion:
2.4.10 Section Headings
